blueprints/node-webkit/files/tests/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I believe only `name` and `main` are needed for this to run in NW.js. We can probably remove the rest.